radish34/api/package.json

```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "build": "cod-scripts build",
     "build:watch": "npm run build -- --watch",
```

Comment:
is `build:watch` redundant if `dev` includes the same with the change?

Reply:
Yes, I will remove